edge/test/integration/scripts/execute.sh

```diff
@@ -17,7 +17,7 @@
 TEST_DIR=$(dirname $(dirname "${BASH_SOURCE[0]}"))
 
 function cleanup() {
-  sudo pkill -9 edgecore || true
```

Comment:
We cannot catch SIGKILL, but we still can use SIGKILL to force shutdown a process.
So, is this necessary to remove "-9" here?

Reply:
Since cannot catch SIGKILL, `reset` would do not graceful shutdown, then would do bad impact on next test.